Added launch icon and set up correct launch names

janitaroyseth opened the pull request:
Because I did flutter clean a lot of files got modified. I added images and image assets in Android Studio and xCode and modified the `info.plist` and `AndroidManifest.XML`. Apps might need reinstalling for the app to work. Please do test.

Commit:
Added launch icon and set up correct launch names

pr0xity reviewed (changes requested):
/project should be removed when moving the app to /solveit :)
![image](https://user-images.githubusercontent.com/16777070/198370731-ef4f49fe-291a-4787-91a1-a5db3b7b8a1c.png)

Commit:
Revert "Added launch icon and set up correct launch names"

This reverts commit a6e5f535f578ce2d08f5a627626b689e18ac2c91.

Commit:
Reverted commit and added launch icons and name anew.

janitaroyseth commented:
Changing project folder name created a lot of merge conflicts with each change. I reverted the commit, it's back to only project folder now, and added the launch icons and settings for appname in android and ios folders.

Commit:
Merge branch 'dev' into 58-set-up-launch-icon-and-launch-with-correct-name

janitaroyseth committed:
Delete .DS_Store

janitaroyseth committed:
Delete .vscode directory

Commit:
Merge branch 'dev' into 58-set-up-launch-icon-and-launch-with-correct-name

pr0xity reviewed (approved):
![image](https://user-images.githubusercontent.com/16777070/199599210-fcb7a769-8abb-4f4d-a47b-1a5075ceaba3.png)
